### DISCUSSION TITLE
v2.4.7-pre.1 is the most stable in a while

### DISCUSSION THREAD

**Discussion by FranciscoBigWings** [WillCodeForCats/solaredge-modbus-multi#459] [General]
Hi @WillCodeForCats 

As my title says, v2.4.7-pre.1 is in my opinion, the most stable release in a while. Congratulations.

I have been testing 2 nights and almost 3 days and it is great. There are no connection issues with inverters, all commands properly change the settings of the inverter and which is more important inverter connectivity hasnt gone unavailable since I updated. Also the battery reports always well and all energy commands related to storage control mode work much better from the first time.

No bug discovered for the moment. So far it is great. And I just wanted to off my hat off to you :)

**Comment by WillCodeForCats** [comment]
[pre.6](https://github.com/WillCodeForCats/solaredge-modbus-multi/releases/tag/v2.4.7-pre.6) will probably be the last one before release unless something else comes up.

**Reply by FranciscoBigWings** [reply]
v2.4.7-pre.7 is working so far so good for me with my two SE and one with LG battery with all energy and storage features being used
